core/domain/classifier_services.py

```diff
@@ -113,7 +113,10 @@ def verify_signature(
 def handle_trainable_states(
     exploration: exp_domain.Exploration,
     state_names: List[str]
-) -> None:
+) -> List[
```

Comment:
Yeah, ok to do that part separately -- I realized that too and that's why I edited the comments, we don't really have a standard to_model() method. Maybe in the future we can.

/cc @vojtechjelinek @aasiffaizal because I think they've been thinking through this sort of refactor as well...